README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 What? Both? There have been probably hundreds of releases for Python, that are still available. The requirement states clearly, that Python 2.7 or higher is required. And I don't see where you have already explained.